opennlp-brat-annotator/pom.xml

```
@@ -17,7 +17,7 @@
 	<parent>
 		<groupId>org.apache.opennlp</groupId>
 		<artifactId>opennlp</artifactId>
-		<version>1.9.3-EBX-2</version>
+		<version>1.9.3-EBX-3</version>
```

We have deployed this version it will need to be bumped
```suggestion
		<version>1.9.3-EBX-4</version>
```

---

Really? Are you sure? I believe that the last version [deployed](https://github.com/ebx/main/pull/8206/files) was 1.9.3-EBX-2 or in the [Apache Archiva](http://repo.ebx.sh/#artifact/org.apache.opennlp/opennlp). But with so many version changes I might have missed something